plugins/maciVoting/services/coordinator/utils/networks.ts

```diff
@@ -0,0 +1,90 @@
+/** This entire file is copied from https://github.com/privacy-scaling-explorations/maci/blob/dev/apps/coordinator/ts/common/networks.ts */
```

Comment:
This file was mainly was used with the `/utils/chain.ts` functions but the network is set and retrievable from the viem, wagmi library which represents the connected wallet to the frontend.

I would say let's just keep `ESupportedNetworks` that are used in other files (I would even suggest moving that enum to utils or something)

Reply:
Didn't even need `ESupportedNetworks` after other removals. Removed this file